ssl/ssl_conf.c

```diff
@@ -671,23 +671,21 @@ static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
             OPENSSL_free(copy);
             return 0;
         }
-        hs_padding = (size_t) atoi(commap + 1);
+        hs_padding = (size_t) strtoul(commap + 1, NULL, 0);
+        if ((hs_padding == ULONG_MAX) && (errno == ERANGE))
+            return 0;
     }
-    block_padding = (size_t) atoi(copy);
+    block_padding = (size_t) strtoul(copy, NULL, 0);
+    if ((block_padding == ULONG_MAX) && (errno == ERANGE))
```

Comment:
Nit: Could you please avoid adding the extra unnecessary parentheses here? Also there should be no space after the cast.